cmd/release-controller/http.go

```diff
@@ -59,6 +59,7 @@ const releasePageHtml = `
 <pre class="ml-4">
 oc patch clusterversion/version --patch '{"spec":{"upstream":"{{ .BaseURL }}graph"}}' --type=merge
 </pre>
+<div class="alert alert-primary">This site is part of OpenShift's continuous delivery pipeline. Neither the builds linked here nor the upgrade paths tested here are officially supported.</br>Please visit the Red Hat Customer Portal for the latest supported product details.</div>
```

Comment:
I would remove the red hat portal part too - someone who comes here and doesn't know about the Red Hat Portal... probably isn't a customer.